src/content/docs/ai-observability/advanced-customization/drop-sensitive-data.mdx

```diff
@@ -0,0 +1,306 @@
+  >
+    **Expression:**
+    ```
+    ([a-zA-Z0-9!#$'*+?^_`{|}~.-]+(?:@|%40)(?:[a-zA-Z0-9-]+\.)+[a-zA-Z0-9-]+)
+    ```
+  </Collapser>
+
+  <Collapser
+    id="indiapanid"
+    title="India PAN ID"
+  >
+    **Expression:**
+    ```
+    ^([a-zA-Z]){5}([0-9]){4}([a-zA-Z]){1}?$
+    ```
+  </Collapser>
+
+  <Collapser
+    id="indiaaadhaar"
+    title="India AADHAAR ID"
+  >
+    **Expression:**
+    ```
+    ^([2-9]{1}[0-9]{3}\s\d{4}\s\d{4})$
+    ```
+  </Collapser>
+
+  <Collapser
+    id="ipv4"
+    title="IP address (ipv4)"
+  >
+    **Expression:**
+    ```
+    ^([0-9]{1,3})\.([0-9]{1,3})\.([0-9]{1,3})\.([0-9]{1,3})$
+    ```
+  </Collapser>
+
+  <Collapser
+    id="japanid"
+    title="Japan national identity number (My number)"
+  >
+    **Expression:**
+    
+    ```
+   (d{4}\sd{4}\sd{4})
+    ```
+  </Collapser>
+
+  <Collapser
+    id="spainnid"
+    title="Spain National ID (NIE/DNI/NIF)"
+  >
+    **Expression:**
+    ```
+    ([a-zA-Z]?[-\s]?\d{7,8}[-\s]?[a-zA-Z])
+    ```
+  </Collapser>
+
+  <Collapser
+    id="ssn"
+    title="US social security number"
+  >
+    **Expression:**
+    ```
+    (\d{3}[-\s\.]?\d{2}[-\s\.]?\d{4})
+    ```
+  </Collapser>
+
+  <Collapser
+    id="uknino"
+    title="UK national insurance number (NINO)"
+  >
+    **Expression:**
+    ```
+    ([a-zA-Z]{2}[-\s]?\d{2}[-\s]?\d{2}[-\s]?\d{2}[-\s]?[a-dA-D])
+    ```
+  </Collapser>
+
+  <Collapser
+    id="usstreetaddress"
+    title="US street address"
+  >
+    **Expression:**
+    ```
+    /\d{1,}(\s{1}\w{1,})(\s{1}?\w{1,})/g
+    ```
+  </Collapser>
+
+  <Collapser
+    id="usphone"
+    title="US phone number"
+  >
+    **Expression:**
+    ```
+    (^[\+]?[1]?[\W]?[(]?[0-9]{3}[)]?[-\s\.]?[0-9]{3}[-\s\.]?[0-9]{4})
+    ```
+  </Collapser>
+
+  <Collapser
+    id="uspassport"
+    title="US passport number"
+  >
+    **Expression:**
+    ```
+    ([a-zA-Z]?\d?\d{5,8})
+    ```
+  </Collapser>
+
+  <Collapser
+    id="dob"
+    title="US Date of birth"
+  >
+    **Expression:**
+    ```
+    ((?:\d{2})?\d\d(?:\\)?(?:\/)?\d\d(?:\\)?(?:\/)?\d{2}(?:\d{2})?)
+    ```
+  </Collapser>
+
+  <Collapser
+    id="ccn"
+    title="Credit card number"
+  >
+    **Expression:**
+    ```
+    ((?:(?:4\d{3})|(?:5[1-5]\d{2})|6(?:011|5[0-9]{2}))(?:-?|\040?)(?:\d{4}(?:-?|\040?)){3}|(?:3[4,7]\d{2})(?:-?|\040?)\d{6}(?:-?|\040?)\d{5})
+    ```
+  </Collapser>
+</CollapserGroup>
+
+## What's next? [#whats-next]
+
+Now that you've secured your customer's data, you can explore AI monitoring:
+
+* [Learn to explore your AI data](/docs/ai-observability/ai-response-performance)
+* Want to adjust your data ingest? [Learn about how to configure AI monitoring](/docs/ai-observability/advanced-customization/adjust-data-ingest). 
+* Did you enable logs? Learn how to [obfuscate sensitive information](/docs/logs/ui-data/obfuscation-ui) from your logs, or [remove entire log messages if they contain sensitive information](/docs/logs/ui-data/drop-data-drop-filter-rules).
```

Comment:
This list has inconsistent punctuation and formatting, honestly it's just the first one that's really a problem. Think you could add a lead-in like the other two have that provide context so it doesn't stick out so much? Just a thought, could just be me.